docker/server-jetty/build.gradle

```diff
@@ -55,6 +60,8 @@ def prepareDocker = project.tasks.register('prepareDocker', Sync) {
 
     from(configurations.serverApplicationDist) {
         into 'serverApplicationDist'
+        // remove -11 suffix, if any
```

Comment:
Better comment _why_ rather than _what_.
And this will replace any instance of `-11` in the string, not just a suffix... though that shouldn't matter for this case.

Reply:
Made the comment and the replace() call clearer.